README.md

```diff
@@ -52,48 +52,36 @@ Refer to our GitHub pages for the course materials and additional resources:
 - [Read](https://opensource.com/education/15/9/open-source-education-israel)
   an article by Laura Novich on [opensource.com](https://opensource.com)
 
-## Requirements
-
-Once we're in the ROSE directory, we need to verify we have pipenv
-installed.  In order to make sure we have pipenv installed:
-
-    pipenv --version
-
-If you don't have it installed, the best way is to install it only for
-your user:
-
-    python -m pip install --user pipenv
-
 ## Getting started
 
 The following commands should be performed only once; after creating the
 environment you will be connecting to the same environment each time you
 open a new session.
 
-Use pipenv to create a virtual environment and to install the rest of
+Use venv to create a virtual environment and to install the rest of
 the dependencies:
 
-    pipenv install
-
-You can also install development packages by running:
-
-    pipenv --dev install
+    python -m venv ./venv
 
 After creating the environment, we want to activate and enter our
 environment (make sure you're in the ROSE directory):
 
-    pipenv shell
+    source ./venv/bin/activate
+
+After entering the virtual enviornment we need to install the project dependencies:
+
+    pip install -r requirments.txt
 
 Indication that you are inside the environment, the prompt line will
 look like this:
 
-    (ROSE) [username@hostname ROSE]$
+    (venv) [username@hostname ROSE]$
```

Comment:
not sure I understand this comment.

Reply:
@nirs before using venv, since we were inside the ROSE dir, pipenv shell generated (ROSE) env as indication.
Changing it to venv shows the relative change for moving to venv.
It won't make the students complicated

Reply:
Trust me keeping the venv in the source will bite you sooner or later.

Reply:
> not sure I understand this comment.

The prompt was changed since you created a virtual enviroment named `venv`. If you create the virtual environment as `ROSE`, you will get the same prompt.